packages/jaeger-ui/src/components/TracePage/index.js

```diff
@@ -278,6 +291,9 @@ export default class TracePage extends React.PureComponent<TracePageProps, Trace
             traceID={traceID}
             onSlimViewClicked={this.toggleSlimView}
             textFilter={textFilter}
+            prevResult={this.prevResult}
+            nextResult={this.nextResult}
```

Comment:
I don't think the use of methods is needed for the scroll props, e.g. 

```jsx
prevResult={this._scrollManager.scrollToPrevVisibleSpan}
```

Reply:
I initially did this to set up the next PR where the buttons move one result at a time, but that will probably just live in the scrollManager too so I'll get rid of the methods.